src/gui/MainWindow.cc

```diff
@@ -2788,16 +2725,7 @@ void MainWindow::animateUpdateDocChanged()
 
 void MainWindow::animateUpdate()
 {
-  if (animate_panel->isVisible()) {
-    bool fps_ok;
-    double fps = this->e_fps->text().toDouble(&fps_ok);
-    if (fps_ok && fps <= 0 && !animate_timer->isActive()) {
-      animate_timer->stop();
-      animate_timer->setSingleShot(true);
-      animate_timer->setInterval(50);
-      animate_timer->start();
-    }
-  }
+    //animateWidget->animateUpdate();
```

Comment:
`MainWindow::animateUpdateDocChanged()` and `MainWindow::animateUpdate()` look to be still connected via the `TabManager`.

Reply:
I look into it.

Reply:
Ah, I see, thanks for the update.